src/gui/src/components/dashboard-grid/index.tsx

```diff
@@ -77,6 +103,9 @@ export const DashboardItem = ({
   )
   const updateQuery = useGraphStore(state => state.updateQuery)
   const updateStamp = useGraphStore(state => state.updateStamp)
+
+  const { savedProjects, saveProject } = useGraphStore()
+
```

Comment:
is there a reason for not sticking with the same standard?

```suggestion
  const saveProject = useGraphStore(state => state.saveProject)
  const savedProjects = useGraphStore(state => state.savedProjects)

```